client/src/components/ParkCard/index.js

```diff
@@ -1,14 +1,28 @@
 import { Card, Accordion, Col } from "react-bootstrap";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCirclePlus, faCircleCheck } from "@fortawesome/free-solid-svg-icons";
+import { useState } from "react";
+const ParkCard = ({ description, name, states, url, images, id }) => {
+  const [parkIcon, setIcon] = useState(faCirclePlus);
```

Comment:
I'm wondering if it would make sense to use one more state object and a ternary that would allow the user to toggle between added and not added. This isn't fully fleshed out, but worked: 
`const [parkAdded, setParkAdded] = useState(false)`

Then inside the addPark function:
`!parkAdded ? setIcon(faCirclePlus): setIcon(faCircleCheck)` - checks to see if the park added state is true or false and sets the appropriate icon.
` setParkAdded(!parkAdded)` - to toggle between the states

This approach did cause a problem with the icon color, but I think that could be figured out somehow.

Reply:
maybe...My initial thought was they would click the plus, the plus would change to a check and the park would be added to that users list of parks (I didn't setup this part yet).  Then if they wanted to remove the park from their list they would do that on the myparks page and that function would change the checkmark back to the plus once they removed it.  This is my understanding if it toggled back and forth- if a user clicked on the plus, it changes to a check-adds it to their list, then they click the check it changes back to the plus, it would also have to then remove it from their list at that point.